libnetwork/sandbox_dns_windows.go

```diff
@@ -18,7 +19,7 @@ func (sb *Sandbox) restoreHostsPath() {}
 
 func (sb *Sandbox) restoreResolvConfPath() {}
 
-func (sb *Sandbox) addHostsEntries(_ context.Context, ifaceIP []string) error {
+func (sb *Sandbox) addHostsEntries(_ context.Context, ifaceIP []netip.Addr) error {
 	return nil
```

Comment:
Still curious if this cannot be implemented on Windows, or if we just never did.